cfgov/cfgov/settings/production.py

```diff
@@ -95,3 +99,75 @@
         "Environment variable ALLOWED_HOSTS is either not defined or is "
         "not valid JSON. Expected a JSON array of allowed hostnames."
     )
+
+# SAML2 Authentication
+#
+# Requires the SAML_AUTH, SAML_ROOT_URL, SAML_ENTITY_ID, and SAML_METADATA_URL
+# environment variables to be defined.
+#
```

Comment:
Worth including a link here [to the docs](https://djangosaml2.readthedocs.io/contents/setup.html#configuration)?

Reply:
Done in e765fc6.